### ISSUE TITLE
Liquibase updateSQL command doesn't generate query for DropAllForeignKeyConstraints change type

### ISSUE THREAD

**Issue by AlexanderSashchenko** [liquibase/liquibase#1951]
## Environment
**Liquibase Version**: 4.3.5

**Liquibase Integration & Version**: CLI, Liquibase test-harness v1.0.1

**Liquibase Extension(s) & Version**: na

**Database Vendor & Version**: any

**Operating System Type & Version**: Windows 10

## Description
Liquibase updateSQL command doesn't generate drop query for DropAllForeignKeyConstraints change type. However in case of applying Liquibase update command changes get executed and all foreign keys get dropped.

## Steps To Reproduce
1) Create two tables and link it with at least two foreign keys within any database from [supported list](https://docs.liquibase.com/change-types/community/drop-all-foreign-key-constraints.html).
2) Run Liquibase updateSQL command with dropAllForeignKeyConstraints changeset.

## Actual Behavior: 
The drop query (Example: ALTER TABLE table_name DROP CONSTRAINT fk_name FOREIGN KEY) is not generated. See the [screenshot](https://monosnap.com/file/QIFK0u98Jvt8tcf27KvCIlWUxJp4Ww).

## Expected/Desired Behavior: 
The drop query is generated.

The database I used for CLI test is Apache Derby embedded v 10.15.2. Same problem was for all databases supported by test-harness v 1.0.1.
[Example of changeLog.xml file.](https://monosnap.com/file/qYKcs7S6BMS7IbTOBicK1AX3qnzjPM)

**Comment by molivasdat** [comment]
Thanks for the excellent description of how to recreate this issue. @AlexanderSashchenko and the screenshots.  We will add this to the list of issues to process.

**Comment by AlexanderSashchenko** [comment]
Update: For DB2 on z/OS described query doesn't get generated AND foreign key don't get dropped in case of running "update" command with DropAllForeignKeyConstraints changeset.

**Comment by AlexanderSashchenko** [comment]
Hi @nvoxland. Yes as we found out with your help the setup of the test was not quite transparent to determine how updateSql shoul work it that case. So to sum up:
I did this test in two separate steps. First I created a table with several foreign keys and after that I run updateSql for dropAllForeignKeys changeSet and it worked as expected.
It this case I propose to close this issue if you agree with me.
